README.md

```diff
@@ -239,3 +239,7 @@ doctrine:
 ## License
 
 This bundle is under the [MIT license](http://opensource.org/licenses/MIT). See the complete license in the file: LICENSE
+
+[GA Image]: https://github.com/gpslab/interval/workflows/CI/badge.svg
+
+[GA Link]: https://github.com/gpslab/interval/actions?query=workflow%3A%22CI%22+branch%3Amaster
```

Comment:
Please explain what these links are?

Reply:
They refer to the state of this build. One is badge and the other one is link to the action "pipeline"

Reply:
Please include the build link and badge at the top of the document as with other badges. Don't use attributes. Save the document formatting style.